sdk/core/azure-core/src/main/java/com/azure/core/annotation/ServiceClient.java

```diff
@@ -4,16 +4,16 @@
 package com.azure.core.annotation;
 
 import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
 import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
 
 /**
  * Annotation given to all service client classes.
  */
-@Retention(RetentionPolicy.SOURCE)
-@Target({TYPE})
+@Retention(RUNTIME)
```

Comment:
Will need to override RevApi due to this change. Additionally, having this annotation retained during runtime does make it part of our public API, before this information wasn't available to the VM but now it can be accessed reflectively

Reply:
Changed this to CLASS and suppressed revapi warning as it won't be accessible during runtime.